Userland/Libraries/LibWeb/DOM/Event.cpp

```diff
@@ -41,6 +41,20 @@ Event::Event(HTML::Window& window, FlyString const& type, EventInit const& event
 {
 }
 
+void Event::visit_edges(Visitor& visitor)
+{
+    visitor.visit(m_target.ptr());
+    visitor.visit(m_related_target.ptr());
+    visitor.visit(m_current_target.ptr());
+    for (auto& it : m_path) {
```

Comment:
Is the `touch_target_list` of PathEntry guaranteed to have its entries in `m_touch_target_list`?

I don't know, and if it's hard to figure out maybe just leaving a FIXME is good enough for now.

Reply:
Oh I missed that one,
looking at `append_to_path`, it seems like it isnt guaranteed